src/content/support/p1-process/index.md

```diff
@@ -0,0 +1,112 @@
+
+![Incident.io Shortcut Popup Screenshot](shortcut_screenshot.png)
+
+In case the incidents comes from an urgent email, [incident.io](https://incident.io/) creates automatically a channel for the incident and pings the person oncall. The incident is created in `triage` so Operations Engineer needs to confirm the severity of the issue before triggering the P1 process. Most of the times our customer provides with a call link to join and confirm the problem.
+
+__Note__: For `triage` incidents, we have designed a [decision workflow](https://incident.io/blog/using-decision-flows) to help engineers when deciding the severity of an incident.
+
+Once the `P1` criticality is confirmed, [incident.io](https://incident.io/) triggers a set of [workflows](https://help.incident.io/en/articles/6971329-getting-started-with-workflows) that helps to drive the incident. The workflows mentioned are:
+
+- `Escalation Matrix` display the different contacts in the customer side to call in case of emergency.
+- `Role assignment` which automatically assign Operation Engineer role to the person reporting the incident.
+- `Ping people oncall` pings the colleagues that are on call once the incident is created automatically by urgent email.
+
+For `P1` incidents the first thing to do is build the team. Most of the time, the engineer creating the channel is not part of the [Incident Coordinators Group](https://giantswarm.app.opsgenie.com/teams/dashboard/f02504a3-83d4-4ea8-b55c-8c67756f9b2e/main), so the engineer will need to escalate the issue to get someone from that team. For that reason, when you create an incident channel, [incident.io](https://incident.io/) shows you a button to escalate and let you select the IC schedule from Opsgenie. In the end, we need at least two person team to manage a critical incident (communications and operations).
+
+![Incident.io Escalate Screenshot](escalate_screenshot.png)
+
+### Investigate
+
+Once the team is built, the person assigned to the Operations Engineer role will carry on with the investigation. The Incident Coordinator will be in contact with the customer, via messaging or in a call, and will provide information to the Operator to help with the investigation.
+
+__Note__: In exceptional occasions the person who acknowledges the alert can manage the communication and fix the problem at the same time, but once it is the case make sure customer is aware of the measures have been implemented to solved the issue.
+
+We leave space for the Operations Engineers to focus on the investigation, but we establish 20 minutes intervals to get back and inform the customer of the current state. Most of the time, the OEs will share the findings in the channel and the IC can pin those messages to help track the actions performed.
```

Comment:
We have 20 minutes intervals here, and then later that incident.io is updating every 30 minutes?

Reply:
yeah let's say 30 minutes